BedrockConflictMetrics.h

```diff
@@ -0,0 +1,68 @@
+#include <libstuff/libstuff.h>
+#include <bitset>
+
+class BedrockConflictMetrics {
```

Comment:
Eh, I started with something more like that and changed it because I thought it was similar. I'm not super passionate about it either, but I'm inclined to keep it like this so I don't risk breaking something while changing it.